### PR TITLE
Update recipe for py312_patch

### PR DESCRIPTION
**Pull request by cbouss**
Preparation for py312 release.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -35,6 +36,7 @@ requirements:
     - python
     - {{ pin_compatible('numpy') }}
     - {{ pin_compatible('scipy') }}
+    - scipy
```
Comment (ryanskeith) [inline recipe/meta.yaml]: Is there a reason why we need both the {{ pin_compatible('scipy') }} and scipy?  Just curious here.

**Review by ryanskeith — APPROVED** [review]
LGTM.  Though I was curious about the scipy entry.

**Review by Jrice1317 — COMMENTED** [review]
LGTM, though I share the same curiosity regarding having both `{{ pin_compatible('scipy') }}` and `scipy`. Is this based on whether `py<312` or `py==312`? If so, should we add that distinction?

File: recipe/meta.yaml (hunk 1 above)
Comment (cbouss) [reply recipe/meta.yaml]: there wasn't, addressed

**Comment by cbouss** [comment]
> LGTM, though I share the same curiosity regarding having both `{{ pin_compatible('scipy') }}` and `scipy`. Is this based on whether `py<312` or `py==312`? If so, should we add that distinction?

That was a mistake, I have removed the extra version.

**Review by Jrice1317 — APPROVED** [review]

**Pull request merged**